### PR TITLE
[Bridges] fix bug in print_active_bridges

### PR DESCRIPTION
**Pull request by odow**
I missed an edge case: if a variable is unsupported and is bridged by a constraint instead.

x-ref https://github.com/jump-dev/JuMP.jl/issues/3301

### CODE REVIEW

File: test/Bridges/debug.jl [hunk 1]
```diff
@@ -277,6 +277,42 @@ function test_print_active_bridges_variable_unsupported()
...
+    (),
+    (),
+    (),
+    (MOI.ScalarAffineFunction,),
+    (),
+    ()
+)
+
+function test_print_active_bridges_variable_bridged_with_constraint()
+    model = MOI.Bridges.full_bridge_optimizer(ConstraintModel{Float64}(), Float64)
```
Comment (odow) [inline test/Bridges/debug.jl:293]:
```suggestion
    model =
        MOI.Bridges.full_bridge_optimizer(ConstraintModel{Float64}(), Float64)
```

**Comment by blegat** [comment]
Yes, that's always a tricky one. There is also the case where it is **not** added with `add_constrained_variables`  so you'll have to apply the constraint bridge even if it's not the shortest path but that doesn't apply to this printing function.

**Review by blegat — APPROVED** [review]
